subworkflows/local/prepare_genome.nf

```diff
@@ -56,8 +56,8 @@ workflow PREPARE_GENOME {
     ch_fasta_fai = Channel.empty()
     if (fasta_fai) ch_fasta_fai = fasta_fai
     if (!(params.fasta_fai) && !('annotate' in step)) {
-        SAMTOOLS_FAIDX(fasta)
-        ch_fasta_fai = SAMTOOLS_FAIDX.out.fai
+        SAMTOOLS_FAIDX(fasta.map{ it -> [[], it]})
```

Comment:
instead of an empty map for meta, can you add some name info like we do for the tabix modules maybe?
```
        TABIX_BGZIPTABIX(target_bed.map{ it -> [[id:it[0].getName()], it] })
```

Reply:
sure, will see if the CI tests pass first